src/system/keyboard.ts

```diff
@@ -98,7 +98,7 @@ export class KeyboardHost {
     const target = getActiveElementOrBody(config.document)
     this.setKeydownTarget(target)
 
-    this.pressed[code] ??= {
+    this.pressed[code] = this.pressed[code] ?? {
```

Comment:
not sure about the `eslint` as that would mean if I understand correclty the `??=` is unnecessary here and we not even need a fallback?